BMM.Core/AppNavigator.cs

```diff
@@ -162,6 +163,9 @@ private async Task RestoreMediaQueue()
                 if (!isAccessTokenValid)
                     await _oidcAuthService.RefreshAccessToken();
 
+                if (_deviceInfo.IsIos)
+                    await Task.Delay(TimeToDetectIfAppOpenedFromDeepLinkInMs);
```

Comment:
does this slow down all startups of the app?

Reply:
I doesn't because recovering queue is a background operation.